src/main/java/tc/oc/pgm/rotation/FixedPGMMapOrderManager.java

```diff
@@ -117,17 +117,14 @@ public void saveCurrentPosition() {
   public void recalculateActiveRotation() {
     int activePlayers = getActivePlayers();
 
-    FixedPGMMapOrder newRotation =
-        rotations.stream()
-            .filter(FixedPGMMapOrder::isEnabled)
-            .map(FixedPGMMapOrder::getPlayers)
-            .sorted()
-            .filter(playerCount -> playerCount >= activePlayers)
-            .map(this::getRotationByPlayerCount)
-            .findFirst()
-            .orElse(null);
-
-    updateActiveRotation(newRotation);
+    rotations.stream()
+        .filter(FixedPGMMapOrder::isEnabled)
+        .map(FixedPGMMapOrder::getPlayers)
+        .sorted()
+        .filter(playerCount -> playerCount >= activePlayers)
```

Comment:
Filter should be applied before sorting, as sorting is a stateful operation that will have to do less work if some rots are filtered out beforehand. Also, should probably not be sorting and just use a `min` reductor

Reply:
Fixed and tested